content/en/docs/setup/best-practices/certificates.md

```diff
@@ -64,18 +64,21 @@ If you don't wish to copy the CA private keys to your cluster, you can generate
 
 Required certificates:
 
-| Default CN                    | Parent CA                 | O (in Subject) | kind                                   | hosts (SAN)                                 |
+| Default CN                    | Parent CA                 | O (in Subject) | kind                                   | hosts (subjectAltName)                      |
```

Comment:
Go with "subject alternative name" if you don't want to abbreviate. subjectAltName is an OID descriptor.